src/imagetilesource.js

```diff
@@ -114,8 +121,8 @@
             }
 
             $.addEvent(image, 'load', function () {
-                _this.width = image.naturalWidth;
-                _this.height = image.naturalHeight;
+                _this.width = $.getNatural(image).width;
+                _this.height = $.getNatural(image).height;
```

Comment:
Can't we just use `image.width` and `image.height` directly?
I am not sure what creating a new Image object in `$.getNatural` changes.